etc/disable-common.inc

```diff
@@ -143,3 +143,8 @@ blacklist ${PATH}/xfce4-terminal
 blacklist ${PATH}/xfce4-terminal.wrapper
 blacklist ${PATH}/mate-terminal
 blacklist ${PATH}/mate-terminal.wrapper
+blacklist ${PATH}/lilyterm
+blacklist ${PATH}/pantheon-terminal
+blacklist ${PATH}/roxterm
+blacklist ${PATH}/roxterm-config
+blacklist ${PATH}/terminix
```

Comment:
How's this?
I felt safer blacklisting urxvtcd as well. :-) If urxvtc is blacklisted, urxvtcd has no use, so why not? That was my reasoning, anyways... :)